Application/EdFi.Ods.SandboxAdmin/Views/Application/Index.cshtml

```diff
@@ -89,21 +79,30 @@
             <div class="modal-body">
                 <fieldset>
                     <div class="control-group">
-                        <label class="control-label" for="txtName1">Application Name:</label>
+                        <label class="control-label" for="txtApplicationName">Application Name:</label>
                         <div class="controls">
-                            <input id="txtName1" type="text" data-bind="value: applicationName, valueUpdate: 'keyup'" class="form-control" placeholder="Name of Application" />
+                            <input id="txtApplicationName" type="text" data-bind="value: applicationName" class="form-control" placeholder="Name of Application" />
                         </div>
                     </div>
                     <div class="control-group">
-                        <label class="control-label" for="txtName2">EducationOrganizationId:</label>
+                        <label class="control-label" for="txtEducationOrganizationId">EducationOrganizationId:</label>
                         <div class="controls">
-                            <input id="txtName2" type="text" data-bind="value: educationOrganizationId, valueUpdate: 'keyup'" class="form-control" placeholder="EducationOrganizationId" />
+                            <input id="txtEducationOrganizationId" type="text" data-bind="value: educationOrganizationId" class="form-control" placeholder="EducationOrganizationId" />
                         </div>
                     </div>
                     <div class="control-group">
-                        <label class="control-label" >Vendor :</label>
+                        
                         <div class="controls">
-                            <select data-bind="options: vendorList, value:selectedVendor, optionsText: 'Name', optionsValue: 'Id'" class="dropdown"></select>
+                            <div class="dropdown">
+                                <label class="control-label" for="txtEducationOrganizationId">Vendor:</label>
+                                <a href="#" class="nav-link dropdown-toggle" id="vendor-dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    Select Vendor
```

Comment:
The selected vendor is not displayed, this can be confusing for the end users
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/43448209/2b3a81ac-4a85-4f53-9d9c-2d1a461242f0)

Reply:
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/34613894/c4688575-6640-4bf7-907e-ec40b0f0f34e)
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/34613894/38c049c1-230d-4929-970b-696cad02af29)